featuretools/tests/primitive_tests/test_primitive_base.py

```diff
@@ -58,9 +58,50 @@ def get_function(self):
 
             def test(x):
                 return x
+
             return test
 
     primitive = TestPrimitive()
     primitive(range(6))
     primitive(range(6))
     assert primitive.get_function_call_count == 1
+
+
+def test_args_string():
+    class Primitive(PrimitiveBase):
+        def __init__(self, bool=True, int=0, float=None):
+            self.bool = bool
+            self.int = int
+            self.float = float
+
+    primitive = Primitive(bool=True, int=4, float=.1)
+    string = primitive.get_args_string()
+    assert string == ', int=4, float=0.1'
+
+
+def test_args_string_default():
+    class Primitive(PrimitiveBase):
+        def __init__(self, bool=True, int=0, float=None):
+            self.bool = bool
+            self.int = int
+            self.float = float
+
+    string = Primitive().get_args_string()
```

Comment:
let's also add a test case where a user does a mix of all the cases we tested individually. 

* change value from default
* uses the default
* doesn't specify an argument 

e.g `Primitive(bool=False, int=0)` in this test case

Reply:
Change applied in [this commit.](https://github.com/Featuretools/featuretools/pull/481/commits/bfb9d689acb3f1bbd6a62354687b8784beae6d64)